src/modules/addTask.js

```diff
@@ -0,0 +1,23 @@
+import {
+  taskInput,
+  createNewTaskElement,
+  incompleteTasksHolder,
+  bindTaskEvents,
+  taskCompleted,
+} from '../taskInput.js';
+import todoItems from './task.js';
+
+const listItem = todoItems;
+const addTask = () => {
+  const listItemName = taskInput.value || 'New Item';
+  todoItems.listItem = createNewTaskElement(listItemName);
+  incompleteTasksHolder.appendChild(todoItems.listItem);
+  bindTaskEvents(todoItems.listItem, taskCompleted);
+  taskInput.value = '';
+  localStorage.setItem('addTask', JSON.stringify(listItemName));
+};
+if (JSON.parse(localStorage.getItem('listItemName')) != null) {
+  listItem.todoItems = JSON.parse(localStorage.getItem('listItemName'));
+}
```

Comment:
- [ ] Kindly in your add method you should save the data in the local storage with the three properties `text, completed, index` for every task.
- [ ] Kindly you should when you try to reload the page you have to display the tasks that you have added, from the local storage.

![image](https://user-images.githubusercontent.com/35535130/160962915-0a736c1d-1f41-48f1-b838-1de7b86d05d9.png)